app/views/homepage/_popular_links.html.erb

```diff
@@ -12,11 +12,11 @@
           text: t("homepage.index.popular_links_heading"),
         } %>
         <ul class="homepage-most-viewed-list" data-module="ga4-link-tracker">
-          <% t("homepage.index.popular_links").each_with_index do | item, index | %>
+          <% publication.popular_links.each_with_index do | item, index | %>
             <li class="homepage-most-viewed-list__item">
               <%= render "govuk_publishing_components/components/action_link", {
-                text: item[:text],
-                href: item[:href],
+                text: item[:title],
+                href: item[:base_path],
```

Comment:
Hi there, MPE team here: the fields that are in the popular links content item are `title` and `url`, not `base_path`.

Reply:
also, example of homepage content from content store. short version
`id: "8a665458-6063-444a-8838-cba7ec05b39a",
  base_path: "/",
  content_id: "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a",
  title: "GOV.UK homepage",
  description: {"value"=>""},
  document_type: "homepage",
  content_purpose_document_supertype: "",
  content_purpose_subgroup: "other",
  content_purpose_supergroup: "other",
  email_document_supertype: "other",
  government_document_supertype: "other",
  navigation_document_supertype: "",
  search_user_need_document_supertype: "",
  user_journey_document_supertype: "finding",
  schema_name: "homepage",
  locale: "en",
  first_published_at: Fri, 26 Feb 2016 11:53:52.000000000 GMT +00:00,
  public_updated_at: Wed, 28 Jun 2023 10:32:34.000000000 BST +01:00,
  publishing_scheduled_at: nil,
  details: {},
  publishing_app: "special-route-publisher",
  rendering_app: "frontend",
  routes: [{"path"=>"/", "type"=>"exact"}],
  redirects: [],
  expanded_links:
   {"organisations"=>
     [{"links"=>{},
       "title"=>"Government Digital Service",
       "locale"=>"en",
       "details"=>
        {"logo"=>{"crest"=>"single-identity", "formatted_title"=>"Government Digital Ser
vice"},
         "brand"=>"cabinet-office",
         "acronym"=>"GDS",
         "default_news_image"=>nil,
         "organisation_govuk_status"=>{"url"=>nil, "status"=>"live", "updated_
at"=>nil}},
       "api_path"=>"/api/content/government/organisations/government-digital-service",
       "base_path"=>"/government/organisations/government-digital-service",
       "withdrawn"=>false,
       "content_id"=>"af07d5a5-df63-4ddc-9383-6a666845ebe9",
       "schema_name"=>"organisation",
       "document_type"=>"organisation",
       "analytics_identifier"=>"OT1056"}],
    "popular_links"=>
     [{"links"=>{},
       "title"=>"Homepage popular links",
       "locale"=>"en",
       "details"=>
        {"link_items"=>
          [{"url"=>"url1.com", "title"=>"title1"},
           {"url"=>"url2.com", "title"=>"title2"},
           {"url"=>"url3.com", "title"=>"title3"},
           {"url"=>"url4.com", "title"=>"title4"},
           {"url"=>"url5.com", "title"=>"title5"},
           {"url"=>"url6.com", "title"=>"title6"}]},
       "api_path"=>nil,
       "base_path"=>nil,
       "withdrawn"=>false,
       "content_id"=>"ad7968d0-0339-40b2-80bc-3ea1db8ef1b7",
       "schema_name"=>"link_collection",
       "document_type"=>"link_collection",
       "public_updated_at"=>"2024-07-16T10:20:03Z",
       "analytics_identifier"=>nil}],
    "level_one_taxons"=>`

Reply:
eg: homepage content under expanded links
expanded_links:
{,
"popular_links"=>
[
{
"links"=>{
},
"title"=>"Homepage popular links",
"locale"=>"en",
"details"=>
{"link_items"=>
[
{"url"=>"url1.com", "title"=>"title1"},
{"url"=>"url2.com", "title"=>"title2"},
{"url"=>"url3.com", "title"=>"title3"},
{"url"=>"url4.com", "title"=>"title4"},
{"url"=>"url5.com", "title"=>"title5"},
{"url"=>"url6.com", "title"=>"title6"}
]},
"api_path"=>nil,
"base_path"=>nil,
"withdrawn"=>false,
"content_id"=>"ad7968d0-0339-40b2-80bc-3ea1db8ef1b7",
"schema_name"=>"link_collection",
"document_type"=>"link_collection",
"public_updated_at"=>"2024-07-16T10:20:03Z",
"analytics_identifier"=>nil
}
],

Reply:
This has now been fixed.